packages/utils/src/bulk-insert-patients.ts

```diff
@@ -154,13 +155,17 @@ function toTitleCase(str: string): string {
     .trim();
 }
 
-function normalizeGender(gender: string | undefined): "M" | "F" {
+function normalizeGender(gender: string | undefined): GenderAtBirth {
   if (gender == undefined) throw new Error(`Missing gender`);
   const lowerGender = gender.toLowerCase().trim();
   if (lowerGender === "male" || lowerGender === "m") {
     return "M";
   } else if (lowerGender === "female" || lowerGender === "f") {
     return "F";
+  } else if (lowerGender === "other" || lowerGender === "un") {
+    return "O";
+  } else if (lowerGender === "unknown" || lowerGender === "unk") {
+    return "U";
```

Comment:
We should also include Metriport's types for these, as this function is used to insert patients populated by our cxs, which will refer to our docs and then should be using `O` and `U` here.